.clj-kondo/src/hooks/metabase/test/data/datasets.clj

```diff
@@ -0,0 +1,56 @@
+(ns hooks.metabase.test.data.datasets
+  (:require [clj-kondo.hooks-api :as hooks]
+            [clojure.set :as set]))
+
+(defn- only-core-drivers?
```

Comment:
New linters to check that we aren't using `^:mb/once` in a namespace with driver tests and that those namespaces are marked `^:mb/driver-tests`